tests/idris2/interactive046/input.idr

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file should probably not exist.

Reply:
Whoops!

Reply:
You've kept it!

Reply:
Argh, sorry! Missed that there was also an `input.idr`